### PR TITLE
[query/service] implement persist_ir

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

File: hail/python/hail/backend/service_backend.py [hunk 1]
```diff
@@ -430,4 +431,9 @@ def register_ir_function(self, name, type_parameters, argument_names, argument_t
         raise NotImplementedError("ServiceBackend does not support 'register_ir_function'")
 
     def persist_ir(self, ir):
-        raise NotImplementedError("ServiceBackend does not support 'persist_ir'")
+        # FIXME: should use context manager to clean up persisted resources
```
Comment (tpoterba) [inline hail/python/hail/backend/service_backend.py:434]: Let's implement this as:
```
fname = TemporaryFilename().name
hl.experimental.write_expression(ir, fname)
return hl.experimental.read_expression(path)
```

Basically the same, but uses globals instead.

**Review by tpoterba — DISMISSED** [review]

**Comment by danking** [comment]
Bump, this should pass now.

**Comment by danking** [comment]
bump

**Comment by tpoterba** [comment]
sorry, though this was in!

**Review by tpoterba — APPROVED** [review]

**Pull request merged**